kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -137,8 +127,8 @@ define([
                 body(
                     {
                         style: {
-                            margin: '0px',
-                            padding: '0px',
+                            margin: '0',
```

Comment:
This gets embedded inside the iframe source, so it won't absorb the css file.

Reply:
Can those just be `0` instead of `'0'`? (not really a showstopper...)